server/src/controllers/christmas-tree.es6

```diff
@@ -393,24 +379,35 @@ const checkPermitValid = permitExpireDate => {
  * @function generateRulesAndEmail - Private function to generate svg, png, and rules html of a permit and send out email
  * @param {Object} permit - permit object
  */
-christmasTree.generateRulesAndEmail = permit =>
-  permitSvgService
-    .generatePermitSvg(permit)
-    .then((permitSvg) => Promise.all([
-      permitSvgService.generatePng(permitSvg),
-      permitSvgService.generateRulesHtml(true, permit),
-    ]))
-    .then(([permitPng, rulesHtml]) => {
-      permit.permitUrl = paygov.createSuccessUrl(permit.christmasTreesForest.forestAbbr, permit.permitId);
-      const rulesText = htmlToText.fromString(rulesHtml, {
-        wordwrap: 130,
-        ignoreImage: true
+christmasTree.generateRulesAndEmail = permit => {
+  return new Promise((resolve, reject) => {
```

Comment:
I don't think you need to create a new promise here. If you keep it then I think you want to either call `reject` or throw an error inside the `catch` block at line 402, otherwise if this happens the promise never resolves or rejects.

Otherwise, If you add `return` in front of `permitSvgService` and return 'sent' on successful email and make sure to rethrow the error after its caught and logged.